package.json

```diff
@@ -15,7 +15,9 @@
     "test": "npx nx run-many --target=test",
     "prepare": "husky install && chmod ug+x .husky/*"
   },
-  "dependencies": {},
+  "dependencies": {
+    "react-hook-form": "^7.51.1"
+  },
```

Comment:
If RHF was installed in `packages/frontend`, I don't think it'd show up in the root level `package.json`

Reply:
It was actually installed in both. My bad. I removed the installation from the root, so it should be good to go.